modules/azure_bing_search_views/src/Plugin/views/filter/AzureBingSearchViewsKeyword.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\azure_bing_search_views\Plugin\views\filter;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\views\Plugin\views\filter\FilterPluginBase;
+
+/**
+ * Simple filter to handle filtering Azure Bing results by query.
+ *
+ * @ingroup views_filter_handlers
+ *
+ * @ViewsFilter("azure_bing_search_views_keyword")
+ */
+class AzureBingSearchViewsKeyword extends FilterPluginBase {
+
+  /**
+   * This filter is always considered multiple-valued.
+   *
+   * @var bool
+   */
+  protected $alwaysMultiple = FALSE;
+
+  /**
+   * Disable the possibility to use operators.
+   *
+   * @var bool
+   */
+  public $no_operator = FALSE;
```

Comment:
Minor.. Could be better to have this lowerCamelCase since this belongs to class.

Reply:
This is not possible to update, check the base class ;)

Reply:
Yes. Could be removed from here ? as initial values are same as base class.